routes.js

```diff
@@ -5,7 +5,7 @@ const path = require('path')
 
 function fastifySwagger (fastify, opts, next) {
   fastify.route({
-    url: '/documentation/json',
+    url: `${opts.prefix}/json`,
```

Comment:
this should not be needed.

Reply:
seems like route prefixing doesn't work when wrapping the plugin with `fastify-plugin`